mobile/lib/modules/asset_viewer/ui/exif_bottom_sheet.dart

```diff
@@ -212,6 +224,72 @@ class ExifBottomSheet extends HookConsumerWidget {
       );
     }
 
+    showPersonNameEditModel(
+      String personId,
+      String personName,
+    ) {
+      return showDialog(
+        context: context,
+        builder: (BuildContext context) {
+          return PersonNameEditForm(personId: personId, personName: personName);
+        },
+      ).then((_) {
+        // ensure the people list is up-to-date.
+        peopleProvider.refresh();
+      });
+    }
+
+    buildPeople() {
+      return people.widgetWhen(

```

Comment:
I just found in #7671 that this `widgetWhen` builder performs terribly in the sliding up panel for some reason. It's very jittery and doesn't let the panel slide up smoothly.

In that PR, I'm refactoring this bottom sheet to use widgets instead of using these `buildThing` functions. One of us should refactor this to use a widget (in this file is fine) instead of this function, and we should avoid using `widgetWhen` and opt to using

```dart
if (people.value == null) {
  return Container();
} else {
  return _PeopleWidget(people: people);
}
```

Or something like that. I'm also happy to just merge this in as-is and fix the performance issue in my own refactor, because this is not a regression.